credentials/src/main/kotlin/web5/sdk/credentials/VerifiablePresentation.kt

```diff
@@ -58,14 +57,14 @@ public class VerifiablePresentation internal constructor(public val vpDataModel:
    * ```
    */
   @JvmOverloads
-  public fun sign(did: Did, assertionMethodId: String? = null): String {
-    val payload = JWTClaimsSet.Builder()
-      .issuer(did.uri)
-      .issueTime(Date())
-      .claim("vp", vpDataModel.toMap())
+  public fun sign(bearerDid: BearerDid, assertionMethodId: String? = null): String {
+    val payload = JwtClaimsSet.Builder()
+      .issuer(bearerDid.uri)
+      .issueTime(Date().time)
```

Comment:
if this is iat may need to change to seconds, currently this is milliseconds I think

Reply:
fixed, and updated the ktdoc